src/components/Reset.js

```diff
@@ -0,0 +1,13 @@
+import { useGameContext } from '../context/GameContext.js';
+
+export default function Reset() {
+  const { setMessage, setBoard, setCatsGame, setActive, board } = useGameContext();
+
+  function handleClick() {
+    setActive(true);
+    setBoard(board.fill(''));
```

Comment:
accessing state inside its setter can get kinda tricky and do weird things sometimes, a more robust solution would be ```setBoard((prevState) => prevState.fill('')```